src/lightning_app/core/constants.py

```diff
@@ -52,19 +52,24 @@ def get_lightning_cloud_url() -> str:
 
 # EXPERIMENTAL: ENV VARIABLES TO ENABLE MULTIPLE WORKS IN THE SAME MACHINE
 DEFAULT_NUMBER_OF_EXPOSED_PORTS = int(os.getenv("DEFAULT_NUMBER_OF_EXPOSED_PORTS", "50"))
```

Comment:
these ENV vars are quite a few at this point, I suggest we prepend them with a prefix (`LIGHTNING`, `LAI` or such).

Reply:
I updated the env var to start with LAI prefix. there's an issue here to update all the other envs: https://github.com/Lightning-AI/lightning/issues/15388